src/test/java/org/codeforamerica/shiba/application/ApplicationRepositoryTest.java

```diff
@@ -196,23 +196,34 @@ void shouldReturnApplicationIdsOfDocumentsToResubmit() {
                 .timeToComplete(Duration.ofSeconds(1))
                 .county(Anoka)
                 .flow(FlowType.FULL)
-                .completedAt(ZonedDateTime.now(UTC).minusDays(2)) // 2 days ago
+                .completedAt(ZonedDateTime.now(UTC).minusDays(2)) // 2 days ago should be included, we no longer limit to last 24 hours
```

Comment:
Could we just simplify this to 
```suggestion
                .completedAt(ZonedDateTime.now(UTC))
```
?